### PR TITLE
Allow updating async migrations config from the UI

### PR DESCRIPTION
**Pull request by yakkomajuri**
Introduces the ability to update constance settings via the API and leverages this to allow updating async migration settings from the UI.

Still mildly rough, handling everything as strings in the frontend, but should be helpful early on. 

These can be updated via the admin panel, but having them here was a suggestion by @tiina303 that I think is useful.

I still need to write some tests and tweak some things - Friday afternoon submission, will pick back up on Monday

Screenshot 2022-01-07 at 16 23 04

### CODE REVIEW

File: bin/plugin-server [hunk 1]
```diff
@@ -37,7 +37,7 @@ if [[ -n $DYNO_RAM ]]; then
   export WORKER_CONCURRENCY=$(( ($DYNO_RAM - 1) / 512 + 1 ))
 fi
 
-python manage.py migrate --check
+python3 manage.py migrate --check
```
Comment (yakkomajuri) [inline bin/plugin-server:40]:
```suggestion
python manage.py migrate --check
```

**Review by tiina303 — APPROVED** [review]
Do the settings set on the UI survive helm upgrades (they should, unless the env is explicitly set in which case maybe change to the env)?

**Comment by yakkomajuri** [comment]
@tiina303 yes - constance stores them in the DB. The default value is looked up from the env but once it is manually updated it should persist that as well afaik

File: bin/tests [hunk 2]
```diff
@@ -28,5 +28,5 @@ PG_HOST="${PGHOST:=localhost}"
 PG_USER="${PGUSER:=posthog}"
 PG_PASSWORD="${PGPASSWORD:=posthog}"
 PG_PORT="${PGPORT:=5432}"
-psql posthog -d "postgres://${PG_USER}:${PG_PASSWORD}@${PG_HOST}:${PG_PORT}" -c "drop database if exists test_posthog"
+psql posthog -c "drop database if exists test_posthog"
```
Comment (yakkomajuri) [inline bin/tests:31]:
```suggestion
psql posthog -d "postgres://${PG_USER}:${PG_PASSWORD}@${PG_HOST}:${PG_PORT}" -c "drop database if exists test_posthog"
```

**Pull request merged**